WordPress/build.gradle

```
@@ -207,7 +207,7 @@ android {
 
             // Limit string resources to Mag16 only for Jetpack
             // Note: this only affects included _locales_; variants for `values-night`, `values-land` and other configs are _still_ preserved in Jetpack and not filtered by this.
-            resourceConfigurations = ["ar", "de", "es", "fr", "he", "id", "it", "ja", "ko", "nl", "pt-rBR", "ru", "sv", "tr", "zh-rCN", "zh-rTW"]
+            resourceConfigurations = ["ar", "de", "es", "fr", "he", "id", "in", "it", "iw", "ja", "ko", "nl", "pt-rBR", "ru", "sv", "tr", "zh-rCN", "zh-rTW"]
```

Maybe also add a note as code-comment about legacy locales?

```suggestion
            // Note 1: this only affects included _locales_; variants for `values-night`, `values-land` and other configs are _still_ preserved in Jetpack and not filtered by this.
            // Note 2: `in` is the legacy locale code for `id` and `iw` is the legacy locale code for `he`. 
            // We need to include those to make sure users with older Android versions and still using those legacy codes still have those translations show up at runtime.
            resourceConfigurations = ["ar", "de", "es", "fr", "he", "id", "in", "it", "iw", "ja", "ko", "nl", "pt-rBR", "ru", "sv", "tr", "zh-rCN", "zh-rTW"]
```

---

That makes sense! I added the comments and also added the legacy codes in the list in a separate line, so it's clear we're adding them as additional requirements (with the comment making that 200% clear).